readthedocs/search/documents.py

```diff
@@ -22,8 +24,19 @@
 log = logging.getLogger(__name__)
 
 
+class RTDDocTypeMixin:
+
+    def update(self, *args, **kwargs):
+        # Hack a fix to our broken connection pooling
+        # This creates a new connection on every request,
+        # but actually works :)
+        log.info('Hacking Elastic indexing to fix connection pooling')
+        self.using = Elasticsearch(**settings.ELASTICSEARCH_DSL['default'])
```

Comment:
What exactly is the fix here? I'm assuming `Elasticsearch` is normally instantiated with a class or instance for the connection pooling?